stablehlo/transforms/StablehloRefineShapes.cpp

```diff
@@ -1060,6 +1072,14 @@ struct UpdateRegionTypePattern : public OpRewritePattern<ReturnOp> {
   }
 };
 
+struct CleanupOperandBufferPattern : public OpRewritePattern<CustomCallOp> {
```

Comment:
I don't understand what this pattern is for. Add a comment?

Reply:
Should be deleted, old cruft. Initially tried to use it as a separate pattern, but refine shapes tries to complete in a single iteration over the program and this relied on pattern ordering, so bundled into the CustomCallOp pattern